Deploy-CAM.ps1

```diff
@@ -2312,9 +2312,14 @@ else {
 
     $chosenSubscriptionNumber = 0 #invalid
     while ( -not (( $chosenSubscriptionNumber -ge 1) -and ( $chosenSubscriptionNumber -le $subscriptionsToDisplay.Length))) {
-        $chosenSubscriptionNumber = 
-        if (($chosenSubscriptionNumber = Read-Host "Please enter the Number of the subscription you would like to use or press enter to accept the current one [$currentSubscriptionNumber]") -eq '') `
-        {$currentSubscriptionNumber} else {$chosenSubscriptionNumber}
+        if( -not $ignorePrompts ) {
+            $chosenSubscriptionNumber = 
```

Comment:
This line won't work

Reply:
Missing back-tick?

Reply:
Actually somehow it did work. I don't know why. But yeah can add the back-tick.